crates/ty_python_semantic/src/types/diagnostic.rs

```diff
@@ -282,6 +349,15 @@ declare_lint! {
     ///
     /// ## Why is this bad?
     /// Classes with an inconsistent MRO will raise a `TypeError` at runtime.
+    ///
+    /// ## Examples
+    /// ```python
+    /// class A: ...
+    /// class B(A): ...
+    ///
+    /// # TypeError: Cannot create a consistent method resolution order
+    /// class C(A, B): ...
+    /// ```
```

Comment:
the example is great. Could we also add a link to https://docs.python.org/3/glossary.html#term-method-resolution-order at the first mention of `MRO` in the "What it does" section?